packages/mocks/README.md

```diff
@@ -16,7 +16,7 @@ This crate is part of the provwasm repository, licensed under the Apache License
 
 // ref: contracts/name/src/msg.rs
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq, JsonSchema)]
```

Comment:
Same comments as attrs/src/msg.rs for query. It should contain the new cw_serde attribute and have attributes for the enum types.

Reply:
done